target/consensource-ui
# Merge internal updates

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

Merge internal changes:

- Feature flags
- Misc CSS fixes

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- Open `.env.feature-flags` and set `ENABLE_TESTING_BANNER=false` and `ENABLE_TESTING_BANNER=true`
- `./docker-helper.sh -r`
- `cd ui && npm run watch`
- Visit http://localhost:8080 and confirm that you see the testing banner
- Visit http://localhost:8080/index_factory.html and confirm that you don't see the `Sign Up` button
- Visit http://localhost:8080/index_factory.html#!/signUp and confirm that you are redirected to the landing page
